languages.toml

```diff
@@ -59,6 +59,7 @@ injection-regex = "toml"
 file-types = ["toml"]
 roots = []
 comment-token = "#"
+language-server = { command = "taplo", args = ["lsp", "stdio"] }
```

Comment:
From the docs ([here](https://taplo.tamasfe.dev/lsp/#native-binary)) it looks like this should be

```suggestion
language-server = { command = "taplo-lsp", args = ["run"] }
```

But maybe I'm just running an out of date binary locally? I'm running `taplo-lsp` 0.2.6 and maybe the most recent taplo has consolidated the LSP and CLI binaries?

Reply:
taplo-lsp can no longer be installed by itself, it can only be run from taplo-cli lsp.
I think it was unified all in one.

https://taplo.tamasfe.dev/lsp/#native-binary
In this section it says install with `cargo install taplo-lsp` but throws an error.

![image](https://user-images.githubusercontent.com/55039048/165584664-3882d4ef-9306-40e4-b401-229a977daad1.png)

I install taplo-cli and it runs with the simple taplo command. I attach a screenshot.

![image](https://user-images.githubusercontent.com/55039048/165585604-d9988200-ba2a-4c94-9d96-1ac3ec06eb84.png)

Reply:
Ah ok, yeah looks like my package manager is behind on taplo versions. This looks good then 👍